x-pack/test/functional/apps/transform/deleting.ts

```diff
@@ -0,0 +1,141 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { TransformPivotConfig } from '../../../../plugins/transform/common/types/transform';
+import { TRANSFORM_STATE } from '../../../../plugins/transform/common/constants';
+
+import { FtrProviderContext } from '../../ftr_provider_context';
+
+function getTransformConfig(continuous?: boolean): TransformPivotConfig {
+  const date = Date.now();
+  return {
+    id: `ec_deleting_${date}_${continuous ? 'continuous' : 'batch'}`,
+    source: { index: ['ft_ecommerce'] },
+    pivot: {
+      group_by: { category: { terms: { field: 'category.keyword' } } },
+      aggregations: { 'products.base_price.avg': { avg: { field: 'products.base_price' } } },
+    },
+    description:
+      'ecommerce batch transform with avg(products.base_price) grouped by terms(category.keyword)',
+    dest: { index: `user-ec_2_${date}` },
+    ...(continuous ? { sync: { time: { field: 'order_date', delay: '60s' } } } : {}),
+  };
+}
+
+export default function ({ getService }: FtrProviderContext) {
+  const esArchiver = getService('esArchiver');
+  const transform = getService('transform');
+
+  describe('deleting', function () {
+    const transformConfigWithPivot = getTransformConfig(false);
+    // const transformConfigWithLatest = getLatestTransformConfig();
+
+    const testDataList = [
+      {
+        suiteTitle: 'transform with pivot configuration',
+        originalConfig: transformConfigWithPivot,
+        expected: {
+          row: {
+            status: TRANSFORM_STATE.STOPPED,
+            mode: 'batch',
+            progress: 100,
+          },
+        },
+      },
+      {
+        suiteTitle: 'continuous transform with pivot configuration',
+        originalConfig: getTransformConfig(true),
+        expected: {
+          row: {
+            status: TRANSFORM_STATE.STARTED,
+            mode: 'continuous',
+            progress: undefined,
+          },
+        },
+      },
+      // TODO enable tests when https://github.com/elastic/elasticsearch/issues/67148 is resolved
+      // {
+      //   suiteTitle: 'delete transform with latest configuration',
+      //   originalConfig: transformConfigWithLatest,
+      //   transformDescription: 'updated description',
+      //   transformDocsPerSecond: '1000',
+      //   transformFrequency: '10m',
+      //   expected: {
+      //     messageText: 'updated transform.',
+      //     row: {
+      //       status: TRANSFORM_STATE.STOPPED,
+      //       mode: 'batch',
+      //       progress: '100',
+      //     },
+      //   },
+      // },
```

Comment:
You shouldn't get issues with the `latest` transform using this dataset, it has been fixed in [this](https://github.com/elastic/kibana/pull/90428) PR

Reply:
Actually, I think there are still some issues, that's why #90428 ultimately didn't activate the `latest` transform clone tests.

Reply:
The PR description has been updated with
```
cloning.ts and editing.ts functional tests are not re-enabled as part of this PR. This will be handled separately.
```
But I can't find that follow-up. Do you know more @darnautov ?

Reply:
@pheyos I recall we haven't enabled tests because of issues with the `EuiDataGrid`. Have you encountered any other problems?

Reply:
@darnautov Unfortunately, the older Jenkins builds from #90428 are not available anymore. But IIRC the latest` transform failed during execution. I will double check and report back here.

Reply:
Ok I'll enable `latest` transform related tests for `7.12` 👌

Reply:
@darnautov I've double checkd and you're right. The latest data update fixed the transform execution issue and now the tests are failing due to a data grid issue.